src/webhooks/handleSubscriptionUpdated.ts

```diff
@@ -14,7 +14,10 @@ export default async function handleSubscriptionUpdated(
   log: FastifyLoggerInstance,
 ): Promise<void> {
   const customerId = subscription.customer as string;
-  const { uuid } = await usersService.findUserByCustomerID(customerId);
+  const { uuid, lifetime } = await usersService.findUserByCustomerID(customerId);
+  if (lifetime) {
```

Comment:
Are we now blocking the subscription update for lifetimes, I understand?

Reply:
We are blocking lifetime update to subscription.